fuse-products/src/main/java/software/tnb/product/junit/SkipOnPodman.java

```diff
@@ -0,0 +1,16 @@
+package software.tnb.product.junit;
+
+import static java.lang.annotation.ElementType.METHOD;
+import static java.lang.annotation.ElementType.TYPE;
+import static java.lang.annotation.RetentionPolicy.RUNTIME;
+
+import org.junit.jupiter.api.condition.DisabledIfEnvironmentVariable;
+
+import java.lang.annotation.Retention;
+import java.lang.annotation.Target;
+
+@Target({TYPE, METHOD})
+@Retention(RUNTIME)
+@DisabledIfEnvironmentVariable(named = "DOCKER_HOST", matches = ".*podman.*", disabledReason = "The test doesn't support Podman")
```

Comment:
this seems a bit fragile, it does what it says it does: skips the test when running on `podman`, but I would expect it skipped the test on all machines that were running `podman`, not only those that had `podman` in their hostname

Reply:
it is based on this configuration: `export DOCKER_HOST=unix://${XDG_RUNTIME_DIR}/podman/podman.sock` [doc](https://java.testcontainers.org/supported_docker_environment/#podman), do you think improving expression like `"unix.*podman.sock"` can be more accurate? any other suggestion? thank you @avano

Reply:
ah ok, I expected the `podman` is part of the machine hostname, in that case disregard what I said and lgtm